modules/smart-agent_systemd-services/README.md

```diff
@@ -0,0 +1,112 @@
+# SYSTEMD-SERVICES SignalFx detectors
```

Comment:
may be `systemd` would be enough and shorter for the module name ?

Reply:
I disagree, today systemd manage A LOT of things.
Beside .service there is .timer, .path, .device, .mount, …

With this module we can only monitor services and I think it should be explicit in the module name.

Reply:
I thought maybe this module could grow up in future to embed more than services but ok no problem just a suggestion